api4/config_test.go

```diff
@@ -90,6 +90,10 @@ func TestGetConfigWithAccessTag(t *testing.T) {
 	t.Run("Can read value with permission", func(t *testing.T) {
 		assert.Equal(t, mockVaryByHeader, cfg.RateLimitSettings.VaryByHeader)
 	})
+
+	t.Run("Contains Feature Flags", func(t *testing.T) {
+		assert.NotNil(t, cfg.FeatureFlags)
+	})
```

Comment:
I feel that we should add few more tests confirming that we indeed do get the feature flags as long as we are some sort of system console admin.

Test 1 is get config as a system user, make sure we don't get it back.
Test 2 is do it as a read only admin, make sure we have feature flags.

Reply:
Test 1 results in a Permissions error, you must have at least one system console read permission, this is already tested in TestGetConfig().
Test 2 is what I added. This is a system user that gets a SYSTEM_CONSOLE_READ permission added to it. So with a single permission we get feature flags.